contributing.md

```diff
@@ -11,7 +11,7 @@ We actively welcome your pull requests.
 
 1. Fork the repo and create your branch from `master`.
 2. If you've added code that should be tested, add tests.
-3. Add an ample description of your additions.

```

Comment:
this is not a typo :)

Reply:
hahaha I thought you were thinking of using 'sample', my bad

Reply:
Back as it was :)